### PR TITLE
Release 2.8.0

### PR DESCRIPTION
**Pull request by tadatuta**
cc @narqo @blond @arikon

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
+
+- New [i18n](https://github.com/bem/bem-core/tree/v2/common.blocks/i18n) block was introduced, providing support for internationalization ([#1074](https://github.com/bem/bem-core/issues/1074)).
+- Now jQuery is included via `https` by default ([#1202](https://github.com/bem/bem-core/issues/1202)).
+- Dependency on `bemhtml-compat` was dropped ([#1186](https://github.com/bem/bem-core/issues/1186)).
```
Comment (narqo) [inline CHANGELOG.md]: Ouch, it doesn't sound like a semver minor change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
+
+- New [i18n](https://github.com/bem/bem-core/tree/v2/common.blocks/i18n) block was introduced, providing support for internationalization ([#1074](https://github.com/bem/bem-core/issues/1074)).
```
Comment (narqo) [inline CHANGELOG.md:7]: We've declared that no new features would be released without documentation. How about docs for `i18n`? Is there a task for that?

File: CHANGELOG.md (hunk 1 above)
Comment (tadatuta) [inline CHANGELOG.md]: There's no need in it in `bem-core` for users of `ENB` anyways.

Users of `bem-tools` who will update to latest `bem-core` version (if any) just need to run `npm i bemhtml-compat` to fix it on their projects. So the change looks quite harmless.

File: CHANGELOG.md (hunk 2 above)
Comment (tadatuta) [inline CHANGELOG.md:7]: Main idea was not to release changes with obsolete docs. But we can decide if it worth being released now or postpone till documentation is ready.

Comment (narqo) [inline CHANGELOG.md:7]: As far as I understand, it's impossible to start using `i18n` on existing projects, with simple update of bem-core. Am I right that one has to change something in the service building process to use it? What kind of changes should be done? What new enb-modules should be added?

I'd prefer to have at least a minimal docs before release, if there are no users who can't wait. Currently, these changes don't feel too useful.

File: CHANGELOG.md (hunk 1 above)
Comment (narqo) [inline CHANGELOG.md]: Wouldn't you mind to add some additional notes about harmlessness of these changes?

File: CHANGELOG.md (hunk 2 above)
Comment (blond) [inline CHANGELOG.md:7]: Now documentation in [enb-bem-i18n](https://github.com/enb-bem/enb-bem-i18n) package.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+- Now jQuery is included via `https` by default ([#1202](https://github.com/bem/bem-core/issues/1202)).
+- Dependency on `bemhtml-compat` was dropped ([#1186](https://github.com/bem/bem-core/issues/1186)).
+
+### Bug fixes
+
+- Bug with undefined handler call in `loader_type_js` was fix ([#1159](https://github.com/bem/bem-core/pull/1159)).
```
Comment (arikon) [inline CHANGELOG.md]: was fixed

File: CHANGELOG.ru.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+- Теперь `jQuery` по умолчанию подключается через `https` ([#1202](https://github.com/bem/bem-core/issues/1202)).
+- Удалена зависимость от `bemhtml-compat` ([#1186](https://github.com/bem/bem-core/issues/1186)).
+
+### В релиз вошли следующие исправления ошибок
+
+- Исправлена ошибка в `loader_type_js`, допускавшая вызовы неопределенного обработчика ([#1159](https://github.com/bem/bem-core/pull/1159)).
+
+### Также в релиз вошли следующие изменения
+
+- Улучшены шаблоны `bem create` для `bemhtml`, `bemtree`, `vanilla.js` и `browser.js` were improved ([#1183](https://github.com/bem/bem-core/issues/1183)).
```
Comment (arikon) [inline CHANGELOG.ru.md]: Лишнее «were improved»

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
```
Comment (arikon) [inline CHANGELOG.md]: Why don't to stop messing up with terminology? Major changes could be only major release according to semver.

File: CHANGELOG.md (hunk 2 above)
Comment (tadatuta) [inline CHANGELOG.md:7]: Added very [basic notes](https://github.com/bem/bem-core/blob/v2/common.blocks/i18n/i18n.ru.md) for now. Hope @innabelaya will improve them later on.

File: CHANGELOG.md (hunk 5 above)
Comment (tadatuta) [inline CHANGELOG.md]: will `Major` -> `Significant` be better?

Comment (narqo) [inline CHANGELOG.md]: How about "Notable changes" as [node.js guys call them](https://github.com/nodejs/node/blob/v4.0.0/CHANGELOG.md)?

Comment (arikon) [inline CHANGELOG.md]: @narqo :+1:

**Comment by tadatuta** [comment]
:up:

**Pull request merged**